internal/client/application/config/config.go

```diff
@@ -0,0 +1,13 @@
+package config
+
+type Config struct {
+	AppKey string `env:"APP_KEY"`
```

Comment:
```suggestion
	appKey string `env:"APP_KEY"`
```
если делаем коннекторы к полю, значит, пытаемся закрыть прямой доступ. А если пытаемся закрыть прямой доступ, то незачем делать поле публичным.

Reply:
хм, немного встал в тупик. Я использую Геттеры для обеспечения контракта между фактическим местом использования конфига и его реализацией. Но, публичные поля в структуре также нужны для возможности парсить в структуру значения по тегам (env, json и тд). Так же изучу этот вопрос более подробнее.

Reply:
Ok, принимается. Не надо ничего менять.